src/theme/index.tsx

```diff
@@ -36,6 +36,19 @@ const BREAKPOINTS = {
   xxxl: 1920,
 }
 
+const transitions = {
+  duration: {
+    slow: 500,
+    medium: 250,
+    fast: 125,
+  },
+  easing: {
```

Comment:
i would call this `timing` or `timingFunction` since you could also have `linear`, `cubic-bezier(n,n,n,n)`, etc. the css attribute is `transition-timing-function`.

Reply:
this makes sense to me